### PR TITLE
Id 518

### PR DESCRIPTION
**Pull request by coldze**

### CODE REVIEW

File: methods/delete.meth [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
```
Comment (mechmind) [inline methods/delete.meth:1]: What is this?

File: gofetcher/gofetcher.go [hunk 2]
```diff
@@ -258,7 +254,7 @@ func parseTimeout(rawTimeout interface{}) (timeout int64) {
 	return timeout
 }
 
-func (gofetcher *Gofetcher) ParseRequest(method string, requestBody []byte) (request *Request) {
+/*func (gofetcher *Gofetcher) ParseRequest(method string, requestBody []byte) (request *Request) {
```
Comment (mechmind) [inline gofetcher/gofetcher.go:257]: You are breaking a package API and keep commented code in same time.

File: app_version.hpp.in [hunk 3]
```diff
@@ -0,0 +1,19 @@
+package version
```
Comment (mechmind) [inline app_version.hpp.in:1]: This package will not be built without cmake and no fallback template file provided. So package cannot be built with plain `go build`.

**Comment by coldze** [comment]
That was a wrong PR, it had to be made on the fork https://github.com/ndk/cocaine-gofetcher.

**Pull request closed without merging**